sdk/attestation/Azure.Security.Attestation/src/AttestationClient.cs

```diff
@@ -16,16 +16,18 @@ namespace Azure.Security.Attestation
     ///
     /// The Attestation client contains the implementation of the "Attest" family of MAA apis.
     /// </summary>
-    public class AttestationClient : IDisposable
+#pragma warning disable CA1001 // Types that own disposable fields should be disposable
+    public class AttestationClient
+#pragma warning restore CA1001 // Types that own disposable fields should be disposable
     {
         private readonly HttpPipeline _pipeline;
         private readonly ClientDiagnostics _clientDiagnostics;
         private readonly AttestationRestClient _restClient;
         private readonly SigningCertificatesRestClient _metadataClient;
         private readonly AttestationClientOptions _options;
         private IReadOnlyList<AttestationSigner> _signers;
+        // NOTE The SemaphoreSlim type does NOT need Disposable based on the current usage.
```

Comment:
Should we mention explicitly that we don't need to do so because `AvailableWaitHandle` isn't referenced, to guard against future mistakes as things evolve?

Reply:
Sure - I didn't know that was the magic that made it safe :).